reflex/components/radix/primitives/progress.py

```diff
@@ -63,22 +66,12 @@ class ProgressIndicator(ProgressComponent):
     color_scheme: Var[LiteralAccentColor]
 
     def _apply_theme(self, theme: Component):
-        global_color_scheme = getattr(theme, "accent_color", None)
-
-        if global_color_scheme is None and self.color_scheme is None:
-            raise ValueError(
-                "`color_scheme` cannot be None. Either set the `color_scheme` prop on the progress indicator "
-                "component or set the `accent_color` prop in your global theme."
-            )
-
-        color_scheme = color(
-            self.color_scheme if self.color_scheme is not None else global_color_scheme,  # type: ignore
-            9,
-        )
+        if self.color_scheme is not None:
+            self.custom_attrs["data-accent-color"] = self.color_scheme
 
         self.style = Style(
             {
-                "background-color": color_scheme,
+                "background-color": color("accent", 9),
```

Comment:
We should make the key `background_color`. Also, I'm a bit unclear on how `color_scheme` plays together with the theme. Is it right that we should always use the theme color, but if color_scheme is set it overrides it? In that case, why do we make it `accent` here?

Reply:
we set the `data-accent-color` attribute on the element to the `color_scheme` provided by the user. and `var(--accent-9)` picks up the "nearest" `data-accent-color`.

this is essentially how all of the existing radix themes components handle overriding the `color`, `radius`, etc props: they don't change the css, instead they set the `data-*` attribute and the CSS tokens respond to that.